pkg/sign/signer.go

```diff
@@ -33,20 +51,26 @@ func Bundle(content Content, keypair Keypair, fulcio *Fulcio, idToken string, ti
 	content.Bundle(bundle, signature, digest, keypair.GetHashAlgorithm())
 
 	// Add verification information to bundle
-	if fulcio != nil {
-		certBytes, err := fulcio.GetCertificate(keypair, idToken)
+	var pubKeyPEM []byte
```

Comment:
nit on naming, can we say "verifierPEM" or something similar? This was one issue I had with Cosign naming, that it perpetuated the use of public keys when certificates are what we prefer.

The naming for the Rekor client LGTM since we do call the verifier a `PublicKey`.